gravitee-apim-console-webui/src/management/api/apis.module.ts

```diff
@@ -1324,6 +1326,20 @@ const states: Ng2StateDeclaration[] = [
     },
     component: ApiV4PolicyStudioDesignComponent,
   },
+  {
+    name: 'management.apis.ng.runtimeLogs',
+    url: '/runtime-logs',
+    data: {
+      apiPermissions: {
+        only: ['api-log-r'],
```

Comment:
Sounds good I'll let you add that in your PR

Reply:
I may not understand how the permission work, but I won't prevent someone without `api-definition-u` from accessing the Runtime Logs as we have a dedicated permission to read the log they should be able to access them 🤔 
However, we can add this check to access the settings state to prevent (maybe that what you were talking about 🙈 )